chore: check rules without filename tag (#27)
coreruleset/crs-linter: README.md

## How does it work

> The script expects multiple arguments to work correctly. For the complete list of possible arguments, please run the script without any argument. You will get some similar output:

Reviewer: ```suggestion
The script expects multiple arguments to work correctly. For the complete list of possible arguments, please run the script without any argument. You will see output similar to the following:
```

```bash
usage: crs-linter [-h] [-o {native,github}] -d DIRECTORY [--debug] -r CRS_RULES -t TAGSLIST [-v VERSION] [-f FILENAME_TAGS_EXCLUSIONS]
crs-linter: error: the following arguments are required: -d/--directory, -r/--rules, -t/--tags-list
```

> #### Arguments review

Reviewer: ```suggestion
#### Arguments overview
```

> * `-h` - gives help

Reviewer: ```suggestion
* `-h` - shows usage information and exits
```

* `-o` - output format, van be `native` (default) or `github`; Note, that `github` format follows [GH](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-a-notice-message) suggestion
* `-d` - directory path to CRS git repository. This is required if you don't add the version.
* `--debug` - show debug information
* `-r` - CRS rules file to check, can be used multiple times, eg `-r ../path/to/crs-setup.conf -r "../path/to/rules/*.conf"`

> * `-t` - path to file which contains list of approved tags; only tags allowed at rules which are listed in this file

Reviewer: ```suggestion
* `-t` - path to file which contains the list of approved tags; tags not listed in this file will be considered check failures when found on a rule
```

If everything is fine, rule returns with 0.

Normally, you should run the script (from `coreruleset` directory):

> ```

Reviewer: ```suggestion
```bash
```

> ../crs-linter/src/crs_linter/cli.py --debug -r crs-setup.conf.example -r 'rules/*.conf' -t util/APPROVED_TAGS -f ../crs-linter/FILENAME_EXCLUSIONS -v "4.13.0-dev"

Reviewer: ```suggestion
../crs-linter/src/crs_linter/cli.py \
  --debug \
  -r crs-setup.conf.example \
  -r 'rules/*.conf' \
  -t util/APPROVED_TAGS \
  -f ../crs-linter/FILENAME_EXCLUSIONS \
  -v "4.13.0-dev"
```

Author: This is a temporary example, we should add `FILENAME_EXCLUSIONS` to CRS too, then we should change here the path.

```

Optionally, you can add the option `--output=github` (default value is `native`):

> ```

Reviewer: ```suggestion
```bash
```

> ../crs-linter/src/crs_linter/cli.py --debug --output=github -r crs-setup.conf.example -r 'rules/*.conf' -t util/APPROVED_TAGS -f ../crs-linter/FILENAME_EXCLUSIONS -v "4.13.0-dev"

Reviewer: ```suggestion
../crs-linter/src/crs_linter/cli.py \
  --debug \
  --output=github \
  -r crs-setup.conf.example \
  -r 'rules/*.conf' \
  -t util/APPROVED_TAGS \
  -f ../crs-linter/FILENAME_EXCLUSIONS \
  -v "4.13.0-dev"
```
